lettuce/simulation.py

```diff
@@ -65,16 +65,14 @@ def __init__(self, flow, lattice, collision, streaming):
     def step(self, num_steps):
         """Take num_steps stream-and-collision steps and return performance in MLUPS."""
         start = timer()
-        if self.i == 0:
-            self._report()
         for _ in range(num_steps):
             self.i += 1
```

Comment:
move this to after the report to recover reporting at `i=0`

Reply:
(however adding a last report seemed problematic, in case step(n) is called several times, then some timesteps would be reported twice)

Reply:
Another option would be to return the reporting to the old placement, eventhough it doesn't agree with the book... then we don't leave the last steps results unreported